webapp/tests/Utils/UtilsTest.php

```diff
@@ -847,4 +847,24 @@ public function testToTsvField()
         $this->assertEquals('team nåme…',   Utils::toTsvField("team nåme…"));
         $this->assertEquals('team🎈name',   Utils::toTsvField("team🎈name"));
     }
+
+    /**
+     * Test Tab Separated Value parsing
+     */
+    public function testParseTsvLine()
+    {
+        $bs  = "\\";
```

Comment:
What is the reason you uses a variable for these two?

Reply:
I thought it was more readable than having to count backslashes, e.g.
```php
Utils::parseTsvLine("team\\\\name\\\\\trank"));
```
vs
```php
Utils::parseTsvLine("team".$bs.$bs."name".$bs.$bs.$tab."rank"));
```

Reply:
Makes sense actually.